package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Sample App for JavaScript SDK view layer for Onfido identity verification",
   "scripts": {
     "predev": "npm run build",
-    "dev": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https",
+    "dev:webpack": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https --cert cert.pem --key key.pem",
```

Comment:
Okay, I misunderstood the flow

I think something like that in the README would go a long way